src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -596,10 +598,10 @@ public static Map<String, String> validateOptions(Map<String, String> options) t
             try
             {
                 long sizeInBytes = FBUtilities.parseHumanReadableBytes(s);
-                if (sizeInBytes < 0)
-                    throw new ConfigurationException(String.format("Invalid configuration, %s should be positive: %s",
+                if (sizeInBytes < targetSSTableSize)
```

Comment:
UCS aims to produce sstables with size between $t / \sqrt 2$ and $t \cdot \sqrt 2$ (More information can be found [here](https://github.com/apache/cassandra/blob/cassandra-5.0/src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.md)).  We should verify that `sizeInBytes > targetSSTableSize * INVERSE_SQRT_2`

Reply:
> UCS aims to produce sstables with size between t/2 and t⋅2 (More information can be found [here](https://github.com/apache/cassandra/blob/cassandra-5.0/src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.md)). We should verify that `sizeInBytes > targetSSTableSize * INVERSE_SQRT_2`
Sorry, I made a mistake in my previous comment.   This is supposed to say `sizeInBytes < targetSSTableSize * INVERSE_SQRT_2`

`min_sstable_size` should be < `targetSSTableSize * INVERSE_SQRT_2`, so if `sizeInBytes > targetSSTableSize * INVERSE_SQRT_2` we should throw an exception.  In this case, we should continue to make sure that `sizeInBytes` is not < 0.  Sorry for the confusion.

Reply:
The purpose of the `min_sstable_size` option is to only split sstables over shards if they are at least as large as `min_sstable_size`.  If it is set, fewer than `base_shard_count` shards will be created to ensure that the estimated size of each sstable is at least `min_sstable_size`, but once there is enough data, the number of shards will be such that the size of each sstable is between $t / \sqrt 2$ and $t \cdot \sqrt 2$ (assuming sstable_growth = 0).  

The goal of this ticket is to verify that `min_sstable_size` is less than the lower bound ($t / \sqrt 2$).